notebooks/official/bigquery-ml/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,1465 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    DEPLOYED_NAME = "bqml-lr-penguins"
</pre><p>Does this need to be unique? If so, use a UUID.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Done.</p>